src/components/tabController/TabBarItem.tsx

```diff
@@ -132,7 +132,7 @@ export default function TabBarItem({
   const sharedSelectedLabelStyle = useSharedValue(JSON.parse(JSON.stringify(selectedLabelStyle)));
 
   useEffect(() => {
-    if (itemWidth.current) {
+    if (props.width) {
```

Comment:
I think this change is unnecessary.

Reply:
I changed it on purpose cause I want it to be clear it depends on the prop being passed. 
While the width ref is kept for later use.